src/test/resources/com/redhat/vertx/pipeline/step/step-part-of-env.yml

```diff
@@ -2,4 +2,4 @@
 - name: step name check
   register: hello
   copy:
-    from: 'Hello from the "{{ step.name }}" step'
+    from: 'Hello from the "{{ stepdef.name }}" step'
```

Comment:
I look at this and I think "Why would it be `stepdef`? I just want the name of the step."